discovery-infra/download_logs.py

```diff
@@ -45,7 +45,7 @@ def main():
     if args.cluster_id:
         cluster = client.cluster_get(args.cluster_id)
         download_logs(client, json.loads(json.dumps(cluster.to_dict(), sort_keys=True, default=str)), args.dest,
-                      args.must_gather, args.update_by_events)
+                      args.must_gather, args.update_by_events, args.pull_secret)
```

Comment:
i think this commit should not be part of this PR. Lets have a little bit smaller scope.

Reply:
The API of `cluster.get_api_vip_from_cluster` has changed.
This function wasn't supposed to be inside cluster.py. On one of the next tasks it will be deprecated and than we will be able to remove it.